configs/apps/ufs/spack.yaml

```diff
@@ -0,0 +1,12 @@
+spack:
+  concretization: separately
+  view: false
+  include:
+  - config.yaml
+  - packages.yaml
+  - modules.yaml
+  - site.yaml
+  - ${SPACK_STACK_DIR}/configs/repos/repos.yaml
+
+  specs:
+  - ufs-bundle-env
```

Comment:
This will install a lot more than what you need to build the ufs (eckit, fckit, atlas, ecbuild, ...) but ok for now, we can fine tune later.